bril-txt/briltxt.py

```diff
@@ -29,16 +30,17 @@
 eop.2: op ";"
 label.1: LABEL ":"
 
-op: IDENT (FUNC | LABEL | IDENT)*
+op: IDENT (FUNC | LABEL | IDENT | SIGNED_INT)*
```

Comment:
This is the concrete-syntax reflection of the abstract-syntax complaint I have above.